src/components/installer/installer.ts

```diff
@@ -0,0 +1,215 @@
+    await mkdir(installFolder);
+    switch (shell.platform()) {
+        case Platform.Windows:
+        {
+            // Instruction say always to use this link for Windows, not to check the version first
+            const kubectlUrl = 'https://storage.googleapis.com/kubernetes-release/release/v1.9.0/bin/windows/amd64/kubectl.exe';
+            const downloadFile = path.join(installFolder, 'kubectl.exe');
+            const downloadResult = await downloadTo(kubectlUrl, downloadFile);
+            if (!downloadResult.succeeded) {
+                return { succeeded: false, result: null, error: [`Failed to download kubectl: ${downloadResult.error[0]}`] };
+            }
+            addPathToConfig(`vs-kubernetes.${tool}-path`, downloadFile);
+            return { succeeded: true, result: null, error: [] };
+        }
+        case Platform.Linux:
+        case Platform.MacOS:
+        {
+            const version = await getStableKubectlVersion();
+            if (!version.succeeded) {
+                return { succeeded: false, result: null, error: version.error };
+            }
+            const os = platformUrlString(shell.platform());
+            const kubectlUrl = `https://storage.googleapis.com/kubernetes-release/release/${version.result.trim()}/bin/${os}/amd64/kubectl`;
+            const downloadFile = path.join(installFolder, 'kubectl');
+            const downloadResult = await downloadTo(kubectlUrl, downloadFile);
+            if (!downloadResult.succeeded) {
+                return { succeeded: false, result: null, error: [`Failed to download kubectl: ${downloadResult.error[0]}`] };
+            }
+            fs.chmodSync(downloadFile, '0777');
+            addPathToConfig(`vs-kubernetes.${tool}-path`, downloadFile);
+            return { succeeded: true, result: null, error: [] };
+        }
+    }
+}
+
+async function getStableKubectlVersion() : Promise<Errorable<string>> {
+    const downloadResult = await download('https://storage.googleapis.com/kubernetes-release/release/stable.txt');
+    if (!downloadResult.succeeded) {
+        return { succeeded: false, result: '', error: [`Failed to establish kubectl stable version: ${downloadResult.error[0]}`] };
+    }
+    const version = fs.readFileSync(downloadResult.result, 'utf-8');
+    fs.unlinkSync(downloadResult.result);
+    return { succeeded: true, result: version, error: [] };
+}
+
+export async function installHelm(shell: Shell) : Promise<Errorable<void>> {
+    const tool = 'helm';
+    const executable = formatBin(tool, shell.platform());
+    const urlTemplate = 'https://storage.googleapis.com/kubernetes-helm/helm-v2.8.2-{os_placeholder}-amd64.tar.gz';
+    return await installToolFromTar(tool, urlTemplate, shell);
+}
+
+export async function installDraft(shell: Shell) : Promise<Errorable<void>> {
+    const tool = 'draft';
+    const urlTemplate = 'https://azuredraft.blob.core.windows.net/draft/draft-v0.11.0-{os_placeholder}-amd64.tar.gz';
```

Comment:
I'd personally hold off until we release v0.12.0 later this week, but it's your call. There is one BIG change to the architecture occurring in that release that might be worth holding off for a few days. :)

https://github.com/Azure/draft/pull/573

If you don't want to wait on that release dropping then I can follow up and PR a v0.12.0 version bump when it's released

Reply:
That makes sense, though I'll be away by then.  Depending on how the review goes I might merge this as is, and then do the much smaller change of updating the version while I'm on vacation.  Thanks for the heads up -- does this also affect the change I made to `Draft: Version` to pass the `-c` flag?  (I added that because installing Draft then immediately doing `Draft: Version` resulted in errors trying to get the server version.)

Reply:
yes. the -c flag on `draft version` was removed in v0.12.0 because there's only the client version:

```
><> draft version -c
Error: unknown shorthand flag: 'c' in -c
```